docs/articles_en/learn-openvino/llm_inference_guide/genai-guide-npu.rst

```diff
@@ -0,0 +1,113 @@
+Run LLMs with OpenVINO GenAI Flavor on NPU
+==========================================
+
+.. meta::
+   :description: Learn how to use the OpenVINO GenAI flavor to execute LLM models on NPU.
+
+This guide provide some extra details on how to utilize NPU with GenAI
+flavor. Please refer to
+:doc:`install OpenVINO GenAI <../../get-started/install-openvino/install-openvino-genai>`
+for more information on OpenVINO GenAI.
+
+Export an LLM model via Hugging Face Optimum-Intel
+##################################################
+
+1. Install the correct components for exporting model:
+   .. code-block:: text
+   
+   # requirements.txt
+   transformers==4.42.4
+   openvino==2024.2.0
+   openvino-tokenizers==2024.2.0
+   nncf==2.11.0
+   optimum-intel @ git+https://github.com/huggingface/optimum-intel.git@439d61f79cf55d5d0b28334f577b6ac3c5ced28f
+   ```
+
+   .. code-block:: console
+
+      pip install -r requirements.txt
+
+2. A chat-tuned TinyLlama model is used in this example. The following conversion & optimization settings are recommended when using NPU:
+
+   .. code-block:: python
+
+      optimum-cli export openvino -m TinyLlama/TinyLlama-1.1B-Chat-v1.0 --weight-format int4 --sym --group-size 128 --ratio 1.0 TinyLlama
+
+Run the generation by using OpenVINO GenAI
+##########################################
+
+1. Install the correct components for running the model on NPU via OpenVINO GenAI:
+   .. code-block:: text
+
+   # requirements.txt
+   openvino==2024.3.1
+   openvino-tokenizers==2024.3.1
+   openvino-genai==2024.3.1
+   transformers==4.42.4
+   torch==2.4.0+cpu
+   diffusers==0.29.2
+   optimum-intel @ git+https://github.com/huggingface/optimum-intel.git@439d61f79cf55d5d0b28334f577b6ac3c5ced28f
```

Comment:
Also, in general case, shouldn't this list of requirements be more relaxed in terms of versions? like just "install the latest"?
Are notebooks (for example) or GenAI samples (for another example) fix the component versions up to specific versions?

Reply:
They're using `>=` rather than `==`.

We may use `>=` for `openvino`,`openvino-tokenizers`,`openvino-genai` for sure as for the transformers and diffusers, probably we may avoid specifying exact versions...

Reply:
Done